modules/nf-core/igvreports/tests/main.nf.test.snap

```diff
@@ -23,6 +35,18 @@
         },
         "timestamp": "2024-03-19T13:55:24.216860945"
     },
+    "testvariantsWithFastaAndTracks - vcf": {
```

Comment:
This still has some of the old snapshots in it, can you regenerate it (with `nf-core modules test igvreports --update`)

Reply:
whoops -- will do.

I am looking at the igvreports github readme again, and it says that the fasta should be indexed. I am going to add that, along with the metadata map to the `fasta` channel input, since that seems to be the current way of things.

Reply:
Yeah, a meta map on the fasta channel is probably sensible, to be consistent with most other tools